ci/create-windows-image.ps1

```diff
@@ -13,13 +13,14 @@
 
 # https://chocolatey.org/docs/how-to-setup-offline-installation
 $CHOCOLATEY_VERSION="0.10.11"
+$DOTNETCORE_SDK_VERSION="2.1.302"
 $GIT_VERSION="2.18.0"
 $RESHARPER_CLI_VERSION="2018.1.2"
 $SPATIAL_VERSION="1.1.9"
 
 # From https://public-cdn.cloud.unity3d.com/hub/prod/releases-win32.json
-$UNITY_VERSION="2018.2.0b10"
-$UNITY_RELEASE_HASH="4bc57476174c"
+$UNITY_VERSION="2018.2.0f2"
+$UNITY_RELEASE_HASH="787658998520"
```

Comment:
somehow my comment vanished so adding it back in: We store these values in two files. Would it be possible to get it from there instead of hardcoding it here

Reply:
Maybe, but I haven't thought of something satisfactory so far. Check out https://github.com/improbable/platform/pull/4575 for more context?